iocage_lib/ioc_json.py

```diff
@@ -360,7 +360,8 @@ def validate_rctl_props(prop, value):
             IOCRCTL.validate_rctl_tunable()
 
             if not re.findall(
-                r'(?:deny|log|devctl|sig\w*|throttle)=\d+(?:b|k|m|g|t|p|)$',
+                r'(?:deny|log|devctl|sig\w*|throttle)='
+                r'\d+(?:[Bb]|[Kk]|[Mm]|[Gg]|[Tt]|[Pp]|)$',
                 value
```

Comment:
why not use `flags=re.IGNORECASE`?

Reply:
Thank you for pointing it out @igalic. It affected actions as well but having a second look confirms that actions are case insensitive. So at the end i think a better approach is doing a lower on the value and keeping the validation as it was originally. Thanks again!